contracts/lootbox-v1/contracts/Library/SaleKindInterface.sol

```diff
@@ -0,0 +1,87 @@
+import "@openzeppelin/contracts/utils/math/SafeMath.sol";
```

Comment:
It won't be compiled I think. The solidity pragma version should be selected here.